Marlin/src/pins/stm32f4/pins_ANET_ET4.h

```diff
@@ -33,6 +33,12 @@
 //
 // EEPROM
 //
+
+/**
+ * Status: Working.
+ * Hardware: AT24C04C (ATMLH744 04CM) 4 Kb => http://ww1.microchip.com/downloads/en/DeviceDoc/AT24C04C-AT24C08C-I2C-Compatible-%20Two-Wire-Serial-EEPROM-4-Kbit-8-Kbit-20006127A.pdf
+ */
+
```

Comment:
Do we really need a comment to say that EEPROM is working? It's defined & setup below.

Reply:
It's nice to know someone tested it. I haven't yet. Now that we know we can trust the author we can remove these comments.